ui/raidboss/popup-text.ts

```diff
@@ -1448,7 +1448,7 @@ export class PopupText {
 
   getDataObject(): RaidbossData {
     let preserveHP = 0;
-    if (this.data && this.data.currentHP)
+    if (this.data.currentHP)
       preserveHP = this.data.currentHP;
```

Comment:
This should possibly be `this?.data.currentHP`? Raidboss is currently breaking on the hosted version because of `this.data` not being defined at this point.

```
[5/15/2022 6:06:37 PM] Info: Raidboss: BrowserConsole: Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'currentHP') (Source: https://quisquous.github.io/cactbot/ui/raidboss/raidboss.bundle.js, Line: 15057)
```